src/docs/reference/support.md

```diff
@@ -107,12 +109,27 @@ Reach out to us at [team@railway.com](mailto:team@railway.com) for more informat
 
 ### Business Class SLOs
 
+We prioritize Business Class customers over all other support requests.
+
 | Severity                             | Acknowledgement Time |
 | ------------------------------------ | -------------------- |
 | P1 (Outages, Escalations)            | One hour - 24/7      |
 | P2 (Bugs)                            | Same Business Day    |
 | P3 (Integrations, General Questions) | Two Business Days    |
 
+For Business Class customers with a shared Slack Connect channel with us, you have access to
+"Critical" urgency level support requests:
+
+<Image
+src="https://res.cloudinary.com/railway/image/upload/v1733325632/docs/cs-2024-12-04-23.20_smvweu.png"
+alt="Screenshot of Slack Account Linking"
+layout="intrinsic"
+width={392} height={255} quality={100} />
+
+This allows you to page our support on-call directly for an immediate response.
+Please only use this for production outages or critical issues preventing your
```

Comment:
critical issues is better, what about critical platform issues?

That way a user will have a harder time justifying a feature request as critical.

Same for the above review comment.